board/app/helpers/html_helper.php

```diff
@@ -10,4 +10,62 @@ function redirect($url)
 	header("Location: " . $url);
 	exit();
 }
-?>
+
+function time_difference($date)
+{
+	$second = 60;
+	$minute = 60;
+	$hour = 24;
+	$yesterday = 1;
+	$week = 7;
+	$date = strtotime($date);
+	$now = strtotime(date("Y-m-d H:i:s"));
+	$today = strtotime(date("Y-m-d"));
+	$day = strtotime(date("Y-m-d", $date));
+	$difference = $now - $date;
+	$time = date("h:ia", $date);
+	$date_difference = ($today - $day) / ($second*$minute*$hour);
+	$time_frame = array($second, $minute, $hour);
+	$time_title = array('second', 'minute', 'hour');
+		
+		if ($date_difference === $yesterday) {
+			return "yesterday at " . $time;
+		} elseif ($date_difference > $yesterday && $date_difference < $week) {
+			return $date_difference . " days ago";
+		}
+
+	$count = count($time_frame);
+		for ($i = 0; $i < $count; $i++) {
+				$difference = $difference / $time_frame[$i];
+					
+					if ($difference > 1) {
+						continue;
+					}
+
+			$num = floor($difference * $time_frame[$i]);
+			$title = $num > 1 ? $time_title[$i] . "s" : $time_title[$i];
+			return isset($num) ? "$num $title ago" : $day;
+		}
+	return date("Y-m-d h:ia", $date);
+}
+
+/*
+* Gets a declared variable from an object and 
+* converts it into an array.
+*/
+function objectToArray($obj)
+{
+    if (is_array($obj)) {
+        return $obj;
+    }
+    if (!is_object($obj)) {
+        return false;
+    }
+    $array = array();
+    foreach ( $obj as $key => $value ) {
+        $array[$key] = $value;
+    }
+    return $array;
+}
```

Comment:
I don't understand the purpose of this function :sweat_smile: 

1. It checks if the passed object is an array, and it returns the object if it is.
2. It checks if the passed object is an object, and it returns false if it is not.
3. Then if it passes the 2 conditions, you create an array, to pass the elements inside the $obj into an associative array?

What are these codes for po? :bow: